deploy/lighthouse/cluster_role.yaml

```diff
@@ -9,11 +9,22 @@ rules:
   resources:
   - services
   - namespaces
+  - endpoints
   verbs:
   - get
   - list
   - watch
   - update
+- apiGroups:
+    - discovery.k8s.io
+  resources:
+    - endpointslices
+  verbs:
+    - create
+    - get
+    - list
+    - watch
+    - update
```

Comment:
```suggestion
    - update
    - delete
```

Reply:
Sorry, missed one small comment. We need delete permissions on lighthouse-agent as we'll be creating/deleting endpoint-slice resource to be distributed for stateful sets.

Reply:
Done,